neuronav/envs/grid_env.py

```diff
@@ -755,35 +760,60 @@ def step(self, action: int):
         if self.stochasticity > self.rng.rand():
             action = self.rng.randint(0, self.action_space.n)
 
+        if self.manual_collect == False:
```

Comment:
Shouldn't this be reversed? If manual collect is false, then collection should be automatic?

Reply:
you are right, it came as a result of me changing the naming convention from auto_collect to manual_collect, which reverses the logic.